klpbuild/klplib/utils.py

```diff
@@ -292,14 +292,16 @@ def get_workdir(lp_name):
     return get_user_path('work_dir')/lp_name
 
 
-def get_datadir():
+def get_datadir(arch=""):
     """
     Get the data directory.
 
     Returns:
         Path: The full path to the livepatch file.
     """
-    return get_user_path('data_dir')
+    if arch: assert arch in ARCHS
```

Comment:
typo? the indentation seems weird :)

Reply:
It's rarely used but you can write `if` statements in one line.
I only use it when the statement is very short.

Otherwise I can also use:
```suggestion
    assert arch in ARCHS if arch else True
```

or the normal way
```suggestion
    if arch: 
        assert arch in ARCHS
```

Let me know which one you prefer :smiley:

Reply:
Personally I would use the second one :)

Reply:
Sure, will change it :smiley: